jiant/tasks/lib/templates/shared.py

```diff
@@ -67,6 +67,21 @@ def double_sentence_featurize(
     feat_spec: FeaturizationSpec,
     data_row_class,
 ):
+    """Apply model- and tokenizer-specific transformations to create a DataRow.
```

Comment:
Maybe specify that this is for two-sentence/two-input tasks.

Reply:
Thanks, addressed w/ 33c7f2b.